User related templates (#1419)
ezsystems/developer-documentation: docs/guide/content_rendering/layout/add_forgot_password.md

# Add forgot password option

> You can add a "forgot password" option to the site front to allow users of a specific SiteAccess, both admin or front, request a password change by customizing the template used in `/user/forgot-password`.

Reviewer: Actually, no. I think the route is available out of the box, but you only customize the template? So maybe something like:
```suggestion
The "forgot password" option allows users of a specific SiteAccess, admin or front, to request a password change.
You can customize the template used in the `/user/forgot-password` route.
```
